core/src/main/java/io/grpc/internal/ClientTransport.java

```diff
@@ -136,12 +136,12 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
 
     /**
      * Invoked when a ping is acknowledged. The given argument is the round-trip time of the ping,
-     * in microseconds.
+     * in nanoseconds.
      *
-     * @param roundTripTimeMicros the round-trip duration between the ping being sent and the
+     * @param roundTripTimeNanos the round-trip duration between the ping being sent and the
      *     acknowledgement received
      */
-    void pingAcknowledged(long roundTripTimeMicros);
+    void pingAcknowledged(long roundTripTimeNanos);
```

Comment:
This is one of the few places that we have allowed ourselves backwards incompatibility, and I think we should be able to exercise it.  People who depend on internal implementation details before GA have implicitly accepted that behavior is not stable.  This change makes our API completely uniform and I would argue is necessary before we publicly commit to never changing it.